### DISCUSSION TITLE
Failed to open /dev/null - open /dev/null: permission denied

### DISCUSSION THREAD

**Discussion by gharik** [concourse/concourse#6861] [Help & Support] [answered]
Hello,

I am trying to follow the Stark & Wayne tutorial: https://concoursetutorial.com/basics/task-hello-world/.

I am using the docker-compose.yml from https://raw.githubusercontent.com/starkandwayne/concourse-tutorial/master/docker-compose.yml on Windows10  (Docker Verison 3.3.0).

When I try to run the first "task_hello_world.yml", the image is pulled but then I get a "permission denied" error, when fly/Docker tries to create or run the container:

```
initializing
selected worker: 48972ac6bfce
selected worker: 48972ac6bfce
waiting for docker to come up...
Pulling busybox@sha256:1ccc0a0ca577e5fb5a0bdf2150a1a9f842f47c8865e861fa0062c5d343eb8cac...
docker.io/library/busybox@sha256:1ccc0a0ca577e5fb5a0bdf2150a1a9f842f47c8865e861fa0062c5d343eb8cac: Pulling from library/busybox
f531cdc67389: Pulling fs layer
f531cdc67389: Verifying Checksum
f531cdc67389: Download complete
f531cdc67389: Pull complete
Digest: sha256:1ccc0a0ca577e5fb5a0bdf2150a1a9f842f47c8865e861fa0062c5d343eb8cac
Status: Downloaded newer image for busybox@sha256:1ccc0a0ca577e5fb5a0bdf2150a1a9f842f47c8865e861fa0062c5d343eb8cac
docker.io/library/busybox@sha256:1ccc0a0ca577e5fb5a0bdf2150a1a9f842f47c8865e861fa0062c5d343eb8cac

Successfully pulled busybox@sha256:1ccc0a0ca577e5fb5a0bdf2150a1a9f842f47c8865e861fa0062c5d343eb8cac.

selected worker: 48972ac6bfce
find or create container on worker 48972ac6bfce: runc run: exit status 1: container_linux.go:367: starting container process caused: process_linux.go:459: container init caused: rootfs_linux.go:121: reopening /dev/null inside container caused: Failed to open /dev/null - open /dev/null: permission denied
errored
```

I have found this older post https://discuss.concourse-ci.org/t/open-dev-null-permission-denied/1923, but setting the CONCOURSE_WORKER_BAGGAGECLAIM_DRIVER does not seem to help

Do you have any idea if there is some additional setup for permissions in Windows needed?

Thank you for your help :-)

**Comment by taylorsilva** [comment, marked as answer]
Hey, there's a bug report for this https://github.com/concourse/concourse/issues/6868
It's an issue with Docker desktop 3.3.0. If you downgrade to 3.2.2 it will work.

**Reply by gharik** [reply]
Thank you for the info 👍 .

**Comment by gharik** [comment]
In case someone encounters the same issue - the problem seems to be fixed in Docker Desktop 3.3.1. So instead of downgrading to 3.2.2, you can now upgrade to 3.3.1 as well :-).
